### PR TITLE
feat(localization): update pseudo translation

### PR DESCRIPTION
**Pull request by jona7150**
Updated translation using Jenkins. Please review and merge @sorenoid 

 CC: @michi6571 @masa6674

### CODE REVIEW

File: toolkit/authentication/src/main/res/values-ar/strings.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  ~
+  --><resources>
+    <string name="server_trust_message">試_The certificate provided by %1$s is not signed by a trusted authority____________________________________驗.</string>
+    <string name="cancel">試_Cancel_____________驗</string>
+    <string name="allow_connection">試_Dangerous: Allow Connection____________________________驗</string>
+    <string name="username_password_login_message">試_Please login to %1$s_____________________驗</string>
+    <string name="username_label">試_Username_________________驗</string>
+    <string name="password_label">試_Password_________________驗</string>
+    <string name="login">試_Login___________驗</string>
+</resources>
```
Comment (gunt0001) [inline toolkit/authentication/src/main/res/values-ar/strings.xml:25]: Could you generate a line break at the end of each file.

**Comment by jona7150** [comment]
Added another commit for pseudo translation updates: 0da60a1

**Comment by jona7150** [comment]
Added another commit for pseudo translation updates: 0859915

**Comment by michi6571** [comment]
@gunt0001 We have fixed the link break issue and created a new commit: https://github.com/Esri/arcgis-maps-sdk-kotlin-toolkit/commit/0859915ab5d133f6470bf2c8ccd535882285931d.  Please review. 

cc @jona7150 @MasaGithub88

**Comment by gunt0001** [comment]
@michi6571 @jona7150 @MasaGithub88 - changes look good to me 👍 

@rchintapalli - please do a second review.

**Review by gunt0001 — APPROVED** [review]

**Review by rchintapalli — APPROVED** [review]

**Pull request merged**